README.md

```diff
@@ -144,6 +144,8 @@ interface GitHub {
   @GET @Path("/repos/{owner}/{repo}/contributors")
   List<Contributor> contributors(@PathParam("owner") String owner, @PathParam("repo") String repo);
 }
+...
```

Comment:
Did you mean this ? Two java code blocks and retaining newlines.
I need to study English more. T.T 

```java
interface GitHub {
  @GET @Path("/repos/{owner}/{repo}/contributors")
  List<Contributor> contributors(@PathParam("owner") String owner, @PathParam("repo") String repo);
}
```
```java
GitHub github = Feign.builder()
                     .contract(new JAXRSModule.JAXRSContract())
                     .target(GitHub.class, "https://api.github.com");           
```